test/Microsoft.Sbom.Targets.Tests/Microsoft.Sbom.Targets.Tests.csproj

```diff
@@ -1,24 +1,42 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net8.0;net472</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net8.0</TargetFrameworks>
```

Comment:
Just for my understanding, does this mean the tests for net472 won't run in the ubuntu/mac pipelines? Is this something we should notify Sarah about?

Reply:
Yes we should notify her, you are correct. .NET Framework doesn't run natively on Ubuntu and Mac OS. Visual Studio seems to work natively on Mac OS though, so yes we should notify her.